apps/documentation/src/routes/documentation/installation.mdx

```diff
@@ -1,14 +1,14 @@
-import { Head } from 'tuono'
 import Breadcrumbs, { Element } from '../../components/breadcrumbs'
 
-<Head>
-  <title>Tuono - Installation</title>
-  <link rel="canonical" href="https://tuono.dev/documentation/installation" />
-</Head>
+import MetaTags from '../../components/meta-tags'
 
-<Breadcrumbs>
-  <Element label="Installation" />
-</Breadcrumbs>
+<MetaTags
+  title="Tuono - Installation"
+  canonical="https://tuono.dev/documentation/installation"
+  description="Tuono is a development environment built in rust and typescript that outputs a website written in both the languages"
+/>
+
+<Breadcrumbs breadcrumbs={[{ label: 'Installation' }]} />
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Don’t put a space after the opening parenthesis. (COMMA_PARENTHESIS_WHITESPACE)
Suggestions: `{`
Rule: https://community.languagetool.org/rule/show/COMMA_PARENTHESIS_WHITESPACE?lang=en-US
Category: TYPOGRAPHY
<!-- __reviewdog__:ChA2Mzg1NzIwNGQxNTc4MzVlEgxMYW5ndWFnZVRvb2w= -->